waku/v2/protocol/filter/server.go

```diff
@@ -298,12 +303,15 @@ func (wf *WakuFilterFullNode) pushMessage(ctx context.Context, peerID peer.ID, e
 		} else {
 			wf.metrics.RecordError(writeResponseFailure)
 		}
-		logger.Error("pushing messages to peer", logging.HexBytes("envelopeHash", env.Hash()), zap.String("pubsubTopic", env.PubsubTopic()), zap.String("contentTopic", env.Message().ContentTopic), zap.Error(err))
+		logger.Error("pushing messages to peer", zap.Error(err))
```

Comment:
We should also log message specific details in error if we want to trace the message isn't it? 
Otherwise logs will be hard to analyze as there maybe interleaving from multiple calls to pushMessage.
Infact we should always log message details in error logs. That way anytime debugging becomes easier as message metadata is logged.

Reply:
Besides the fields added to the `logger` in lines 270 to 275, what other metadata do you think it would be useful to add?

Reply:
Those fields should be sufficient. 
Ah, my bad. I mistook logger being appended with info log.